verification/curator-service/ui/src/components/App.tsx

```diff
@@ -267,8 +276,12 @@ export default function App(): JSX.Element {
         setCreateNewButtonAnchorEl,
     ] = useState<Element | null>();
     const [selectedMenuIndex, setSelectedMenuIndex] = React.useState<number>();
+    const [linelistSearchQuery, setLinelistSearchQuery] = React.useState<
+        string
```

Comment:
you don't need the <string> here if you provide a default value from which the type can be inferred

Reply:
Done.